### ISSUE TITLE
"Allow-Ranges" should be "Accept-Ranges"

### ISSUE THREAD

**Issue by SteffenDE** [mar10/wsgidav#152]
https://github.com/mar10/wsgidav/blob/08768f6c4364021a3b1f4da8866960469e7e78a9/wsgidav/request_server.py#L1469

I think this should be "Accept-Ranges", I do not find any information about an "Allow-Ranges" header online.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Ranges

**Comment by mar10** [comment]
Btw., since you seem to have analyzed the request logs and protocol quite a bit:
Do you also test with FileExplorer and MiniRedirector and have experienced the problemx described in #149 ?

**Comment by SteffenDE** [comment]
I recently tested a fresh install of Windows 10 and the performance seemed fine (compared to Windows 7 which is unusably slow and refuses to mount shares with Basic Auth).
I'll do some more testing with Windows clients when I have the time, but Windows is definitely no priority for me.
Every other client I tested worked fine: macOS Finder, Cyberduck, iOS FileBrowser App, nextcloud external storage. Therefore I'd say that this is a Windows issue.
